bpf/lib/conntrack.h

```diff
@@ -231,10 +231,10 @@ static inline int __inline__ ct_lookup6(void *map, struct ipv6_ct_tuple *tuple,
 			if (skb_load_bytes(skb, l4_off + 12, &flags, 2) < 0)
 				return DROP_CT_INVALID_HDR;
 
-			if (unlikely(flags.rst || flags.fin))
-				action = ACTION_CLOSE;
-			else
+			if (unlikely(flags.syn && !flags.ack))
```

Comment:
Just curious: Should this be `if (likely(flags.syn && !flags.ack))` ? Not sure what's the likelihood of SYN occurring :)

Reply:
SYN is set and ACK is not set only on the very first packet a TCP connection, so only a fraction of TCP packets has this set of flags, hence the unlikely().

Reply:
@jrajahalme Thanks for the explanation!